Mathlib/Data/Finset/Pointwise.lean

```diff
@@ -2062,8 +2074,65 @@ theorem card_dvd_card_mul_right {s t : Finset α} :
     ((· • t) '' (s : Set α)).PairwiseDisjoint id → t.card ∣ (s * t).card :=
   card_dvd_card_image₂_right fun _ _ => mul_right_injective _
 
+@[to_additive (attr := simp)]
+lemma inv_smul_finset_distrib (a : α) (s : Finset α) : (a • s)⁻¹ = op a⁻¹ • s⁻¹ := by
+  ext; simp [← inv_smul_mem_iff]
+
+@[to_additive (attr := simp)]
+lemma inv_op_smul_finset_distrib (a : α) (s : Finset α) : (op a • s)⁻¹ = a⁻¹ • s⁻¹ := by
+  ext; simp [← inv_smul_mem_iff]
+
 end Group
 
+section SMulWithZero
+variable [Zero α] [Zero β] [SMulWithZero α β] [DecidableEq β] {s : Finset α} {t : Finset β}
```

Comment:
Why do you move this section?

Reply:
Moved and tagged `zero_smul_subset` simp. I need it to fire in my new proofs.

Reply:
`zero_smul_subset` is not `simp`. Which lemma do you mean?

Reply:
Whoops yes sorry